src/WebJobs.Script/Extensions/FileUtility.cs

```diff
@@ -282,5 +282,19 @@ private static void DoSafeAction(Action action, bool ignoreErrors)
             {
             }
         }
+
+        public static async Task MarkContainerDisabled()
```

Comment:
These new methods don’t really belong in the FileUtility class. Can we move this to a more appropriate type?

Reply:
Moved to Utility.cs